config/index.js

```diff
@@ -48,6 +48,9 @@ const config = {
     rollbar: {
       accessToken: process.env.ROLLBAR_ACCESS_TOKEN,
       environment: process.env.ROLLBAR_ENV
+    },
+    albumsApi: {
+      url: 'https://jsonplaceholder.typicode.com/albums'
```

Comment:
Add a environment variable here ,maybe with a dafault value like  ` process.env.API_URL || 'url'`

Reply:
And `/albums` should be added by services, think if you want use `/photos` later :eyes: